Vagrantfile

```diff
@@ -1,108 +0,0 @@
-    stackstorm.vm.synced_folder "../arteria-provisioning/", "/arteria/arteria-provisioning"
-
-    stackstorm.vm.provider "virtualbox" do |v|
-      v.memory = 2048
-      v.cpus = 2
-      v.customize ["modifyvm", :id, "--ioapic", "on"]
-    end
-
-    stackstorm.vm.provision "ansible" do |ansible|
-      ansible.playbook = "ansible-st2/playbooks/arteriaexpress.yaml"
-      ansible.inventory_path = "ansible-st2/inventories/test_inventory"
-    end
-
-  end
-
-  # Configure the biotank stand-in
-  config.vm.define "testarteria1" do |testtank|
-
-    testtank.vm.box = "puppetlabs/centos-6.6-64-puppet"
-
-    testtank.vm.provider "virtualbox" do |v|
-      v.memory = 8000
-      v.cpus = 4
-      v.customize ["modifyvm", :id, "--ioapic", "on"]
-    end
-
-    testtank.vm.hostname = "testarteria1"
-    testtank.vm.network :private_network, ip: '192.168.42.43'
-
-    testtank.vm.synced_folder "../arteria-packs/", "/arteria/arteria-packs"
-    testtank.vm.synced_folder "../arteria-core/", "/arteria/arteria-core"
-    testtank.vm.synced_folder "../arteria-bcl2fastq/", "/arteria/arteria-bcl2fastq"
-    testtank.vm.synced_folder "../arteria-siswrap/", "/arteria/arteria-siswrap"
-    testtank.vm.synced_folder "../arteria-runfolder/", "/arteria/arteria-runfolder"
-    testtank.vm.synced_folder "../arteria-provisioning/", "/arteria/arteria-provisioning"
-    #testtank.vm.synced_folder "/data/arteria_test_data/", "/data/testarteria1/runfolders"
-
-    testtank.vm.provision "ansible" do |ansible|
-      ansible.playbook = "ansible-st2/playbooks/arteriaexpress.yaml"
-      ansible.inventory_path = "ansible-st2/inventories/test_inventory"
-    end
-
-  end
-
-  # Configure the uppmax stand-in
-  config.vm.define "testuppmax" do |testuppmax|
-
-    testuppmax.vm.box = "puppetlabs/centos-6.6-64-puppet"
-    testuppmax.vm.hostname = "testuppmax"
-    testuppmax.vm.network :private_network, ip: '192.168.42.44'
-
-    testuppmax.vm.provision "shell", inline: "mkdir -p /tmp/runfolders"
-    testuppmax.vm.provision "shell", inline: "chown -R vagrant:vagrant /tmp/runfolders"
-    testuppmax.vm.provision "shell", inline: "mkdir -p /tmp/summaries"
-    testuppmax.vm.provision "shell", inline: "chown -R vagrant:vagrant /tmp/summaries"
-
-    testuppmax.vm.provision "shell", inline: 'echo export SNIC_RESOURCE=milou >> /home/vagrant/.bash_profile'
-    testuppmax.vm.provision "shell", inline: "printf '#!/bin/bash\necho $@\n' > /usr/local/bin/sbatch"
-    testuppmax.vm.provision "shell", inline: "chmod +x /usr/local/bin/sbatch"
-
-    testuppmax.vm.provision "ansible" do |ansible|
-      ansible.playbook = "ansible-st2/playbooks/arteriaexpress.yaml"
-      ansible.inventory_path = "ansible-st2/inventories/test_inventory"
-    end
-  end
-
-  # Deploy ssh keys to all host to ensure they have the
-  # password less ssh-access to each other.
-
-  vagrant_ssh = "/home/vagrant/.ssh"
-
-  config.vm.provision "file",
```

Comment:
Action point: Make sure we have password-less ssh connections between machines in prod. #51